vendor/assets/stylesheets/spree/frontend/global.css.scss

```diff
@@ -2365,13 +2524,13 @@ a.btn.btn-block {
 	display: block;
 }
 
-.social-nav a.facebook-icon {
-	background-image: image-url("facebook.svg");
-}
+// .social-nav a.facebook-icon {

```

Comment:
please uncomment this line. if its creating problem then discuss it with umer/me.